single_number/single_number.py

```diff
@@ -4,8 +4,12 @@
 '''
 def single_number(arr):
     # Your code here
-
-    pass
+    for elem in arr:
+        if arr.count(elem) < 2:
+            return elem
+        
+    return False
+        
```

Comment:
This is a really nice solution! There's something sneaky about this solution though, that makes it `O(n^2)` for a run-time complexity.

`arr.count` actually iterates through the array just like a for-loop does. So if you count that as `O(n)` and the outer for-loop as `O(n)`, you get `O(n*n)` or `O(n^2)`.

Since the objective for moving forward with this project is optimization, see if you can find another solution that reduces the run-time to `O(n)`. If you need any hints let me know!

This is still an otherwise, very nice solution!